monkey/common/agent_configuration/agent_sub_configurations.py

```diff
@@ -151,8 +151,12 @@ class NetworkScanConfiguration(MutableInfectionMonkeyBaseModel):
         :param targets: Configuration for targets to scan
     """
 
-    tcp: TCPScanConfiguration = Field(title="TCP scanner")
-    icmp: ICMPScanConfiguration = Field(title="Ping scanner")
+    tcp: TCPScanConfiguration = Field(
+        title="TCP scanner", description="Configure TCP scanning options"
+    )
+    icmp: ICMPScanConfiguration = Field(
+        title="Ping scanner", description="Configure ICMP scanning options"
+    )
     fingerprinters: Tuple[PluginConfiguration, ...] = Field(
         title="Fingerprinters",
         description="Fingerprint modules collect info about external "
```

Comment:
Suggested change: 
Fingerprint modules that collect info about external services

Reply:
I think we want to describe what fingerprinter plugins are. My suggestion is to do "Fingerprint plugins identify network services found".